--- src/modules/module_utils.h.orig	2009-12-06 15:04:41.794575028 +0100
+++ src/modules/module_utils.h	2009-12-06 15:04:46.456583006 +0100
@@ -421,7 +421,7 @@
 
 
 /* So that gcc doesn't complain */
-int getline(char**, size_t*, FILE*);
+//int getline(char**, size_t*, FILE*);
 
 pthread_mutex_t module_stdout_mutex;
 
--- src/modules/module_main.c.orig	2009-12-06 15:05:04.837354699 +0100
+++ src/modules/module_main.c	2009-12-06 15:05:13.346311782 +0100
@@ -22,7 +22,7 @@
  */
 
 /* So that gcc doesn't comply */
-int getline(char**, size_t*, FILE*);
+//int getline(char**, size_t*, FILE*);
 
 #define PROCESS_CMD(command, function) \
 if (!strcmp(cmd_buf, #command"\n")){ \
