--- speech-dispatcher-0.6.6/src/python/Makefile.in~	2008-02-04 14:51:34.000000000 +0100
+++ speech-dispatcher-0.6.6/src/python/Makefile.in	2008-02-23 16:46:00.161824116 +0100
@@ -5,7 +5,7 @@
 
 install:
 	if test -x /usr/bin/python; then \
-		./setup.py install --prefix=$(prefix); \
+		./setup.py install --prefix=$(prefix) --root=$(DESTDIR) ; \
 	else \
 	echo "Python not found, not installing speechd module"; \
 	fi
