--- speech-dispatcher-0.7.1/config/speechd.conf.orig	2010-09-10 10:23:54.000000000 +0200
+++ speech-dispatcher-0.7.1/config/speechd.conf	2012-04-19 18:22:31.531394055 +0200
@@ -206,7 +206,7 @@
 #  - configuration is the path to the config file of this module,
 #    either relative (to etc/speechd/modules/) or absolute
 
-AddModule "espeak"       "sd_espeak"   "espeak.conf"
+#AddModule "espeak"       "sd_espeak"   "espeak.conf"
 AddModule "festival"     "sd_festival"  "festival.conf"
 #AddModule "flite"        "sd_flite"     "flite.conf"
 #AddModule "ivona"	 "sd_ivona"    "ivona.conf"
